ddtrace/internal/writer/writer.py

```diff
@@ -670,7 +670,7 @@ def start(self):
             # appsec remote config should be enabled/started after the global tracer and configs
             # are initialized
             if os.getenv("AWS_LAMBDA_FUNCTION_NAME") is None and (
-                config._appsec_enabled or asbool(os.getenv("DD_REMOTE_CONFIGURATION_ENABLED", "true"))
+                config._appsec_enabled or config._remote_config_enabled
             ):
                 from ddtrace.appsec._remoteconfiguration import enable_appsec_rc
 
```

Comment:
Not related to this PR but I noticed now this lines are redundant with:

https://github.com/DataDog/dd-trace-py/pull/6681/files#diff-8fe127132688091917c9fe6cf1d184bc1eb2b7d4eb4aa4a21379ec520ea2b0f8L257